### PR TITLE
C++ operators and ability to set default nbits & es.

### PR DESCRIPTION
**Pull request by xman**

### CODE REVIEW

File: lib/posit.cpp [hunk 1]
```diff
@@ -8,6 +8,16 @@
 
 using namespace std;
 
+int Posit::default_nbits = 32;
```
Comment (libcg) [inline lib/posit.cpp]: this needs to be camelcase. `Posit::sDefaultNbits`

File: lib/posit.cpp [hunk 2]
```diff
@@ -8,6 +8,16 @@
...
+int Posit::default_es = 2;
+
+
```
Comment (libcg) [inline lib/posit.cpp]: remove newline

File: lib/posit.h [hunk 3]
```diff
@@ -10,53 +10,75 @@ class Posit {
...
+    float getFloat() const;               // n = x
+    double getDouble() const;             // n = x
 
     // debug
     void setBits(POSIT_UTYPE bits);
     POSIT_UTYPE getBits();
     void print();
 };
 
+
```
Comment (libcg) [inline lib/posit.h]: remove newline

File: lib/posit.cpp (hunk 1 above)
Comment (libcg) [reply lib/posit.cpp]: same with `Posit::sDefaultEs`

**Comment by xman** [comment]
Thanks. Didn't see the default nbit and e. Will redo again and send in pull request.

**Pull request closed without merging**